src/js/components/PlacementConstraintsPartial.js

```diff
@@ -102,26 +78,21 @@ export default class PlacementSection extends Component {
         "column-6": hideValueColumn
       };
 
-      if (index === 0) {
-        fieldLabel = placementConstraintLabel(
-          "Field",
-          "If you enter `hostname`, the constraint will map to the agent node hostname. If you do not enter an agent node hostname, the field will be treated as a Mesos agent node attribute, which allows you to tag an agent node.",
-          { isRequired: true }
-        );
-        operatorLabel = placementConstraintLabel(
-          "Operator",
-          "Operators specify where your app will run.",
-          { isRequired: true }
-        );
-      }
-      if (index === 0 && !hideValueColumn) {
-        valueLabel = placementConstraintLabel(
-          "Value",
-          "Values allow you to further specify your constraint.",
-          { linkText: "Learn more", isRequired: hasOneRequiredValue }
-        );
+      if (isFirstConstraint) {
+        fieldLabel = this.getPlacementConstraintLabel("Field");
+        operatorLabel = this.getPlacementConstraintLabel("Operator");
+        valueLabel = this.getPlacementConstraintLabel("Value");
       }
 
+      const fieldValue = (
+        <FieldInput
+          name={`constraints.${index}.value`}
+          type="text"
+          value={constraint.value}
+          disabled={!!valueIsRequiredEmpty}
```

Comment:
The type coercion is unnecessary because this value comes from `requiresEmptyValue()` which always returns boolean.

Reply:
Not always boolean, during my tests I remember where was `undefined` for that reason because of the `disabled` attribute I decided to keep it safe.

Reply:
Okay, cool makes sense 👍